etcd-manager/Makefile

```diff
@@ -20,6 +20,13 @@ IMAGE_BASE                 := $(STABLE_DOCKER_REGISTRY)/$(STABLE_DOCKER_IMAGE_PR
 BAZEL?=bazelisk
 BAZEL_FLAGS=--features=pure --platforms=@io_bazel_rules_go//go/toolchain:linux_amd64
 
+#PASSWORD=$(shell cat /home/leila/.scw/hashicorp/skey)
```

Comment:
I would suggest migrating that in the configuration of your run target in the IDE